recipe/meta.yaml

```diff
@@ -26,9 +26,8 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - yasm               # [win32]
+    - nasm
     - perl
-    - make               # [unix]
```

Comment:
can we keep make as a build dependency on unix?  The only reason it is not need is that make is included in the docker image, using the conda provided version is much preferred.

Reply:
oh, sure, didn't know that, I erroneously supposed it was brought in by the compiler directive